str/trtools/merge_str_runner.py

```diff
@@ -49,43 +44,31 @@ def main(
         dataset, list(external_wgs_ids)
     )
 
-    vcf_input = []
-    if caller == 'eh':
-        for id in list(external_id_to_cpg_id.values()):
-            sample_vcf_file = b.read_input_group(
-                vcf=os.path.join(input_dir, f'{id}_eh.reheader.vcf.gz'),
-                tbi=os.path.join(input_dir, f'{id}_eh.reheader.vcf.gz.tbi'),
-            )
-            vcf_input.append(sample_vcf_file.vcf)
-
-    elif caller == 'gangstr':
-        for id in list(external_id_to_cpg_id.values()):
-            sample_vcf_file = b.read_input_group(
-                vcf=os.path.join(input_dir, f'{id}_gangstr.vcf.gz'),
-                tbi=os.path.join(input_dir, f'{id}_gangstr.vcf.gz.tbi'),
-            )
-            vcf_input.append(sample_vcf_file.vcf)
-    else:
-        raise ValueError('Invalid caller')
-    multi_vcf_file_path_string = ','.join(str(vcf_path) for vcf_path in vcf_input)
-
     trtools_job = b.new_job(name='mergeSTR')
     trtools_job.image(TRTOOLS_IMAGE)
     trtools_job.storage('20G')
     trtools_job.cpu(8)
+    trtools_job.cloudfuse(f'cpg-{dataset}-main', '/vcffuse')
+
+    vcffuse_path = []
+    for id in list(external_id_to_cpg_id.values()):
+        vcf = os.path.join(input_dir, f'{id}_eh.reheader.vcf.gz')
+        suffix = vcf.removeprefix('gs://').split('/', maxsplit=1)[1]
+        vcffuse_path.append(f'/vcffuse/{suffix}')
+    vcffuse_path = ','.join(vcffuse_path)  # string format for input into mergeSTR
 
     trtools_job.declare_resource_group(ofile={'vcf': '{root}.vcf'})
 
     trtools_job.command(
         f"""
      
-    mergeSTR --vcfs {multi_vcf_file_path_string} --out {trtools_job.ofile} --vcftype {caller}
+    mergeSTR --vcfs {vcffuse_path} --out {trtools_job.ofile} --vcftype eh
```

Comment:
Don't you want to write to the `cloudfuse` bucket here directly instead of using `ofile`?